backend/src/app.ts

```diff
@@ -21,6 +22,14 @@ mongoose.connect(mongoURI, {}, () => {
   console.log("Connected to Database.");
 });
 
+const pairing = new Pairing({
```

Comment:
This will save everytime the server runs creating duplicate copies so let's remove this

Reply:
Yeah I forgot to remove that oop